django/db/models/base.py

```diff
@@ -660,6 +660,25 @@ def _set_pk_val(self, value):
                 setattr(self, parent_link.target_field.attname, value)
         return setattr(self, self._meta.pk.attname, value)
 
+    def _validate_force_insert(self, force_insert):
+        if isinstance(force_insert, bool) or not force_insert:
+            return force_insert
```

Comment:
Could we switch the problem around and have `force_insert=True` turn into `(cls,)` and `force_insert=False` turn into `()`. Truthyness would be preserved and type checks for `force_insert` in `_save_parents` could be avoided.